umaserverdemo/src/main/kotlin/com/lightspark/plugins/HTTP.kt

```diff
@@ -20,6 +22,8 @@ fun Application.configureHTTP() {
             Json {
                 prettyPrint = true
                 isLenient = true
+                ignoreUnknownKeys = true
+//                explicitNulls = false
```

Comment:
Are you intentionally committing this? Or just something you were messing around with locally?

Reply:
Thanks for the catch! I meant to uncomment this before merging. Was testing something. Sending a followup now.